### ISSUE TITLE
Error installing captured on Mac OS X 10.5: uninitialized constant FileUtils

### ISSUE THREAD

**Issue by ghost** [csexton/captured-ruby#6]
```
rainman :: ~ » captured --install                                                                                                                                                                                                                                        
Installing captured
/Library/Ruby/Gems/1.8/gems/captured-0.4.1/bin/captured:83: uninitialized constant FileUtils (NameError)
    from /usr/bin/captured:19:in `load'
    from /usr/bin/captured:19
```

**Comment by csexton** [comment]
What version of ruby and ruby gems do you have installed?

**Comment by ghost** [comment]
```
rainman :: ~ » ruby -v
ruby 1.8.6 (2009-06-08 patchlevel 369) [universal-darwin9.0]

rainman :: ~ » gem -v
1.3.7
```

**Comment by thunfischbrot** [comment]
Experiencing the same problem here.

```
$ sudo captured --install
Installing captured
/Library/Ruby/Gems/1.8/gems/captured-0.4.1/bin/captured:83: uninitialized constant FileUtils (NameError)
from /usr/bin/captured:19:in `load'
from /usr/bin/captured:19

$  ruby -v
+ruby 1.8.7 (2009-06-12 patchlevel 174) [universal-darwin10.0]
$ gem -v
1.6.2
```

**Comment by thunfischbrot** [comment]
Can be fixed by adding
    require 'fileutils.rb'
in the top of /bin/captured.

Attention: Running the install with sudo gives
    launchctl: Dubious ownership on file (skipping): /Users/[your username]/Library/LaunchAgents
because it tries to access your LaunchAgents as root. Instead run the install without sudo. If it fails try fixing the permissions with the Disk Utility.

**Comment by csexton** [comment]
Thanks for the catch thunfischbrot. I will go ahead and add the require fileutils to the bin, and I guess I could supplement the documentation to explicitly say that you should not run `captured --install` with sudo. 

If you ran it with sudo I would recommend manually removing `~/Library/LaunchAgents/com.codeography.captured` and `~/.captured.yml` and re-running `captured --install`

The fileutils error should be fixed in v 0.4.2.
